server/src/main/scala/org/http4s/server/middleware/CSRF.scala

```diff
@@ -0,0 +1,187 @@
+package org.http4s.server.middleware
+
+import java.nio.charset.StandardCharsets
+import java.security.{MessageDigest, SecureRandom}
+import java.time.Clock
+import java.util.Base64
+import javax.crypto.spec.SecretKeySpec
+import javax.crypto.{KeyGenerator, Mac, SecretKey}
+
+import cats.Applicative
+import cats.data.{Kleisli, OptionT}
+import cats.effect.Sync
+import cats.syntax.all._
+import org.http4s.headers.{Cookie => HCookie}
+import org.http4s.server.HttpMiddleware
+import org.http4s.util.{CaseInsensitiveString, encodeHex}
+import org.http4s._
```

Comment:
What editor are you using?  See if you can get all the imports to sort for this project.  In IntelliJ, I think you just delete all the groups.  (I'd still like to figure out how to check this in as a style for the IntelliJ folks.)